submission/proxy.go

```diff
@@ -44,11 +44,13 @@ const (
 var (
 	proxyOnce      sync.Once
 	logListUpdates monitoring.Counter
+	rspLatency     monitoring.Histogram // ep => value
 )
 
 // proxyInitMetrics initializes all the exported metrics.
 func proxyInitMetrics(mf monitoring.MetricFactory) {
 	logListUpdates = mf.NewCounter("log_list_updates", "Number of Log-list updates")
+	rspLatency = mf.NewHistogram("http_latency", "Latency of responses in seconds", "ep")
```

Comment:
What does this metric tell you that `http_log_latency` doesn't?

Reply:
It tells you the response latency for multiplexed calls (according to policy). Those include several submission calls and possible additional delay if some submissions were not successful.

Reply:
It might be helpful to put something to that effect in the description string.